Epam.TestAutomation.Web/PageObjects/Pages/MainPage.cs

```diff
@@ -0,0 +1,22 @@
+using Epam.TestAutomation.Core.BasePage;
+using Epam.TestAutomation.Web.PageObjects.Panels;
+using OpenQA.Selenium;
+
+namespace Epam.TestAutomation.Web.PageObjects.Pages;
+
+public class MainPage : BasePage
+{
+    public override string Url => "https://www.epam.com/";
+
+    public FooterBlock FooterBlock => new FooterBlock(By.XPath("//*[@class='footer section']"));
+
+    public HeaderBlock HeaderBlock => new HeaderBlock(By.XPath("//*[@class = 'header__content']"));
+
+    public By searchButton = (By.XPath("//*[@class='header-search__button header__icon']"));
```

Comment:
What is the reason of creating fields of locators?

Reply:
To use BrowserFactory.Browser.FindElement(mainPage.SearchButton); construction